### PR TITLE
feat(type): new v11 specs

### PR DESCRIPTION
**Pull request by sstrubberg**
Closes [11085](https://github.com/carbon-design-system/carbon/issues/11085)

Typography > Type Specs UI

Page needed to be updated to reflect [v11 specs](https://v11.carbondesignsystem.com/guidelines/typography/type-sets)

#### Testing
- Navigate to the Typography > Type Specs UI page and see if the Type Sets are like the one from the v11 site.

### CODE REVIEW

File: src/pages/typography/type-specs-ui.mdx [hunk 1]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
 a Productive set designed primarily for product use and an Expressive set which
 adds a series scales and fluid behavior.
 
-<TypesetStyle
-  title="Productive"
-  breakpointControls={false}
-  typesets="supportingStyle,body,fixedHeading"
-/>
+## Utility styles
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:223]: In the context of these pages I think these headers could be H3s
```suggestion
### Utility styles
```

File: src/pages/typography/type-specs-ui.mdx [hunk 2]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
...
-  typesets="fluidHeadings,FluidParagraphsAndQuotes,fluidDisplay"
-/>
+The utility styles are for use with productive and expressive moments and
+include styles for code snippets, labels for captions and helper text, as well
+as legal copy. Productive styles have a suffix of `-01` and expressive styles
+have a suffix of `-02`.
+
+<TypesetStyle typesets="smallStyle" />
+
+## Body styles
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:232]:
```suggestion
### Body styles
```

File: src/pages/typography/type-specs-ui.mdx [hunk 3]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
...
+
+There are two body styles for productive and expressive moments. Productive
+styles have a suffix of `-01` and expressive styles have a suffix of `-02`.
+
+<TypesetStyle typesets="body" />
+
+## Fixed heading styles
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:239]:
```suggestion
### Fixed heading styles
```

File: src/pages/typography/type-specs-ui.mdx [hunk 4]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
...
+The fixed heading styles are used for product pages where multiple containers
+are used and space efficiency is key. Fixed means they are not responsive. The
+type size remains constant regardless of break point.
+
+Creators of web pages also use the fixed headings `-01` and `-02` where smaller
+headings are needed.
+
+<TypesetStyle typesets="fixedHeadings" />
+
+## Fluid heading styles
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:250]:
```suggestion
### Fluid heading styles
```

File: src/pages/typography/type-specs-ui.mdx [hunk 5]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
...
+
+The fluid heading styles are primarily used in web pages, and are therefore part
+of the expressive set of type styles. These headings are responsive and the type
+styles change size at different breakpoints.
+
+Do not use these styles inside a container. They may be used in product pages
+where text sits outside of a container, and a blend of expressive and productive
+type styles is desired for hierarchy and distinction. For more information, see
+[Type usage](/guidelines/typography/type-usage).
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:259]: Outdated link, its actually a [v11 link](https://v11.carbondesignsystem.com/guidelines/typography/styling-strategies) so it wont work yet.
```suggestion
[Type usage](https://carbondesignsystem.com/guidelines/typography/styling-strategiese).
```

File: src/pages/typography/type-specs-ui.mdx [hunk 6]
```diff
@@ -192,17 +192,56 @@ needs and how type works across various screen sizes. The IBM type set includes
...
+<br />
+
+<TypesetStyle typesets="fluidHeadings" breakpointControls={true} />
+
+## Fluid display styles
+
+The callout and display styles are part of the expressive set and being fluid,
+they will adjust at different breakpoints. Do not use these styles inside a
+container. For guidance about using display styles, see
+[Type usage](/guidelines/typography/type-usage#expressive-use-cases/).
```
Comment (aagonzales) [inline src/pages/typography/type-specs-ui.mdx:270]: Broken link, current [V11 link](https://v11.carbondesignsystem.com/guidelines/typography/styling-strategies#expressive-use-cases)

```suggestion
[Type usage](https://carbondesignsystem.com/guidelines/typography/styling-strategies#expressive-use-cases).
```

**Review by aagonzales — CHANGES_REQUESTED** [review]

**Review by aledavila — APPROVED** [review]
lgtm

**Review by jnm2377 — APPROVED** [review]

**Pull request merged**